### ISSUE TITLE
Text Input with typing feedback (iOS) doesn't work

### ISSUE THREAD

**Issue by goderbauer** [flutter/flutter#10599]
Characters and Words are not read back when setting is enabled.

/cc @cbracken

**Comment by goderbauer** [comment]
This is now a dup of https://github.com/flutter/flutter/issues/12786.
